unicodetools/data/ucd/dev/EastAsianWidth.txt

```diff
@@ -2612,10 +2612,17 @@ FFFD           ; A  # So         REPLACEMENT CHARACTER
 1FA60..1FA6D   ; N  # So    [14] XIANGQI RED GENERAL..XIANGQI BLACK SOLDIER
 1FA70..1FA7C   ; W  # So    [13] BALLET SHOES..CRUTCH
 1FA80..1FA88   ; W  # So     [9] YO-YO..FLUTE
+1FA89          ; W  # So         HARP
```

Comment:
Not critical, but is there a reason not to merge these additions with existing ranges?

Reply:
I had forgotten to regenerate the UCD.
The new CI check complained about that, too.